### ISSUE TITLE
Nav bar not hiding on 16.1

### ISSUE THREAD

**Issue by hung0702** [Magisk-Modules-Repo/HideNavBar#82]
Hey, great mod. I just installed the April update on my Pixel 6 and v12.0 was working fine. I upgraded to HideNavBar v13.0 and the nav bar reappeared. I went on telegram to try 16.1 but not success there either. I removed the mod, rebooted, then installed v12.0 again and the nav bar is still there (but terminal app is not). So I just uninstalled and tried a navbar remover from someone else that others have confirmed working (but fewer features) and it also does not work.

Is there a way to revert any changes made by the terminal app? There must be a ghost in the machine right? After flashing, v12 was working, but updating to v13+ broke the star feature of this mod. Is there something else I should try? 

![mgk](https://user-images.githubusercontent.com/16511272/162606743-c4b6082f-68b0-48f4-ab5c-5269a43653e0.png)
 
FYI waterfall28 changes the status bar height, but I suspect that doesn't matter because I toggled it off and the nav bar was still there.

**Comment by hung0702** [comment]
Ok I think it has something to do with the keyboard height. I reinstalled again and the nav bar was hidden. I tried changing the kb height and it reappeared

**Comment by DanGLES3** [comment]
I've still yet to been able to point out the source of this issue as it seems quite isolated to few cases

**Comment by DanGLES3** [comment]
You may try version 12.0 and see if it works out for you, if it does the issue is likely caused by buggy aapt support in your ROM (which i can't do much to solve)

**Comment by undermark5** [comment]
To be fair, one would think that Google would have proper aapt support in the stock pixel roms, so, I'm not certain that would be the above would apply, (though I'm also not experienced with any of what this module is doing so take my thoughts with a very large grain of salt)

**Comment by dd805bb** [comment]
Android 14 beta, I can confirm v12 works hiding pill and v13+ does not hide pill.

**Comment by DanGLVK** [comment]
> Android 14 beta, I can confirm v12 works hiding pill and v13+ does not hide pill.

I honestly don't even know, by all means the fabricated overlays used on v20 and up should work on anything A12+ as it circumvents the potential issue of overlay APKs getting rejected (which happened in all previous versions for some users)

Did you try the latest version?

https://github.com/Magisk-Modules-Alt-Repo/HideNavBar/releases/tag/v21

**Comment by dd805bb** [comment]
Yeah tried the latest version. I am on Android 14 Beta. It flashes normally but doesn't work. Tried a few different options just to see if I could get any to work. No dice. Gonna switch to Magisk debug and see what I come up with. I think the overlayfs isn't hooking because of the new implementation. It boots fine. No bootloop. I did try v12 and that actually worked. If that helps.

**Comment by undermark5** [comment]
Are you sure you actually tried the latest version? Do you have v13 or v21? This repo is not the active one anymore (can it be archived?).

**Comment by dd805bb** [comment]
Yes, I am using v21.

**Comment by dd805bb** [comment]
![Screenshot_20230416-180235](https://user-images.githubusercontent.com/41106849/232355181-1c614ed3-3c59-4257-90d5-8e54860ae544.png)
Here it installed with the pill still showing. I only commented on this after following the other one here. I didn't have issues on 13 qpr. But this is a clean Beta 14 install.
